gateway/1.5/modules/ROOT/pages/policies-pdk-prerequisites.adoc

```diff
@@ -1,5 +1,105 @@
-= Policy Development Kit Prerequisites
+= Reviewing Policy Development Kit Prequisites
```

Comment:
As far as I can tell, there is no such word as Prequisites. It is Prerequisites. I see that you changed it her and in the file name in nav.adoc.